src/uniforms/buffer.rs

```diff
@@ -19,6 +22,15 @@ pub struct TypelessUniformBuffer {
     buffer: BufferAny,
 }
 
+impl<T: Copy> GlObject for UniformBuffer<T> {
```

Comment:
`<T: ?Sized + Content>`

The bounds should simply be the same as what the main object requires.